backend/check_stac_metadata/utils.py

```diff
@@ -161,6 +145,20 @@ def save(self, key: str) -> None:
                 multihash=asset["multihash"],
             ).save()
 
+    def validate_directory(self, url: str, parent_metadata_url: str) -> None:
+        root_path = get_url_before_filename(self.traversed_urls[0])
+        if root_path != get_url_before_filename(url):
+            self.validation_result_factory.save(
+                url,
+                Check.MULTIPLE_DIRECTORIES,
+                ValidationResult.FAILED,
+                details={
+                    "message": f"“metadata file: {parent_metadata_url} links to {url}”"
```

Comment:
- I suspect the typographic quotes should be around the two URLs, not this whole line.
- Do we want to generally capitalize the first word in these error messages?
- Should there be a colon in this line?